src/Microsoft.VisualStudio.ProjectSystem.Managed/PublicAPI.Shipped.txt

```diff
@@ -180,4 +180,7 @@ virtual Microsoft.VisualStudio.ProjectSystem.Properties.InterceptingPropertyValu
 virtual Microsoft.VisualStudio.ProjectSystem.Properties.InterceptingPropertyValueProviderBase.OnSetPropertyValueAsync(string! propertyName, string! unevaluatedPropertyValue, Microsoft.VisualStudio.ProjectSystem.Properties.IProjectProperties! defaultProperties, System.Collections.Generic.IReadOnlyDictionary<string!, string!>? dimensionalConditions = null) -> System.Threading.Tasks.Task<string?>!
 Microsoft.VisualStudio.ProjectSystem.Debug.ILaunchProfile2
 Microsoft.VisualStudio.ProjectSystem.Debug.ILaunchProfile2.EnvironmentVariables.get -> System.Collections.Immutable.ImmutableArray<(string! Key, string! Value)>
-Microsoft.VisualStudio.ProjectSystem.Debug.ILaunchProfile2.OtherSettings.get -> System.Collections.Immutable.ImmutableArray<(string! Key, object! Value)>
\ No newline at end of file
+Microsoft.VisualStudio.ProjectSystem.Debug.ILaunchProfile2.OtherSettings.get -> System.Collections.Immutable.ImmutableArray<(string! Key, object! Value)>
+virtual Microsoft.VisualStudio.ProjectSystem.Properties.InterceptingPropertyValueProviderBase.IsValueDefinedInContextAsync(string! propertyName, Microsoft.VisualStudio.ProjectSystem.Properties.IProjectProperties! defaultProperties) -> System.Threading.Tasks.Task<bool>!
+Microsoft.VisualStudio.ProjectSystem.Properties.IInterceptingPropertyValueProvider2
+Microsoft.VisualStudio.ProjectSystem.Properties.IInterceptingPropertyValueProvider2.IsValueDefinedInContextAsync(string! propertyName, Microsoft.VisualStudio.ProjectSystem.Properties.IProjectProperties! defaultProperties) -> System.Threading.Tasks.Task<bool>!
```

Comment:
Just checking... you didn't manually change this, right? I had added a script to add this stuff several months back. Just seeing if that is working properly. 😊

Reply:
correct!